### ISSUE TITLE
Add project to metric scope in Cloud Monitoring (MonitoringProjects resource) 

### ISSUE THREAD

**Issue by kirankrishn** [GoogleCloudPlatform/k8s-config-connector#633]
**Describe the feature or resource**
For Config Connector resources we are not able to find the KCC Elements for Adding 
    1. Adding Project to Metric Scope in Cloud Monitoring
    2. Private Service Connection
    3. Cross Project Binding
    4. Network Intelligence Centre Connectivity Test

**Importance**
   Blocker.
   We are implementing a Foundation Build for a Customer using Config Connector and above Blueprints are required. Customer is using GitLab landing zone and doesn't want any manual changes in the environment.

**Comment by caieo** [comment]
Hi @kirankrishn, can you provide some more information about this issue? I'm not sure if you meant to submit this issue without any context.

**Comment by caieo** [comment]
@kirankrishn: 

> 1. Adding Project to Metric Scope in Cloud Monitoring

The next release should include this resource (https://cloud.google.com/monitoring/api/ref_v3/rest/v1/locations.global.metricsScopes.projects). Please let me know if that is the same resource you're describing here.

> 2. Private Service Connection

I am not a specialist with setting up private service connections, but if I understand correctly, Config Connector should be able to support all the resources required to establish a private service connection. [ServiceNetworkingConnection](https://cloud.google.com/config-connector/docs/reference/resource-docs/servicenetworking/servicenetworkingconnection) + various Compute/SQL resources. We also have this sample that includes a private IP example: https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/samples/resources/sqlinstance/private-ip-instance

> 3. Cross Project Binding

Can you elaborate what you're requesting here? Are you asking for cross-project IAMPolicy Binding? Or asking about referencing resources from other projects? If you're asking about cross-project resources references, you should be able to accomplish that from using [`external` in the resource reference](https://cloud.google.com/config-connector/docs/how-to/creating-resource-references#external_references).  

>4. Network Intelligence Centre Connectivity Test

Is this the resource you're asking for support for? https://cloud.google.com/network-intelligence-center/docs/connectivity-tests/reference/networkmanagement/rest/v1/projects.locations.global.connectivityTests 

Let me know if I'm misinterpreting your requests. If these are in fact separate requests, please file separate issue requests for each.

**Comment by kirankrishn** [comment]
Hey,


  1.  Adding Project to Metric Scope in Cloud Monitoring - yes, This is the resource that we are looking into.
  2.  Private Service Connection - We are investigating on this resource and will get back soon on this.
  3.  Cross Project Binding - We had a discussion with ***@***.******@***.***> we had provided couple of links to check the resource and we will check and get back on this.
  4.  Network Intelligence Centre Connectivity Test - I have created another Issue with Google for the same request, please find it - Config Connector Resource for Network Intelligence Centre Connectivity Test #639

Config Connector Resource for Network Intelligence Centre Connectivity Test * Issue #639 * GoogleCloudPlatform/k8s-config-connector (github.com)<https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/639>


Thanks & Regards,
Kiran K

**Comment by caieo** [comment]
@kirankrishn, I'm going to rename this bug to represent your request your first ask (Project in metric scope in Cloud Monitoring). For the follow up asks, please file separate bugs after checking with the team. Thank you for filing issue #639 as a separate issue. 

In the future, if possible, please refrain from attaching the whole email thread in your message responses -- it's pretty hard to parse through. It also looks like the emails are blurred out, so I'm not entirely sure who you reached out to with regards to this comment: "3.  Cross Project Binding - We had a discussion with ***@***.******@***.***> we had provided couple of links to check the resource and we will check and get back on this."

**Comment by caieo** [comment]
@kirankrishn, `MonitoringMonitoredProject` should be the resource you're looking for and is now supported in 1.79.0 (https://github.com/GoogleCloudPlatform/k8s-config-connector/releases/tag/v1.79.0)
